### ISSUE TITLE
Accessibility for error indicator

### ISSUE THREAD

**Issue by dyachera** [vaadin/framework#11632]
Currently in components that have the ability to display an error indicator, it is not accessible by keyboard as of Vaadin 8.8.2. Would it be possible to allow these to be navigated to by keyboard when visible so they can be read by assistive technology?

**Comment by dyachera** [comment]
.
